mscore/inspector/inspectorElementBase.cpp

```diff
@@ -26,15 +26,17 @@ InspectorElementBase::InspectorElementBase(QWidget* parent)
       {
       e.setupUi(addWidget());
       e.offset->showRaster(true);
-      iList = {
+
+      const std::vector<InspectorItem> iiList = {
             { Pid::VISIBLE,   0, e.visible,    e.resetVisible   },
             { Pid::Z,         0, e.z,          e.resetZ         },
             { Pid::COLOR,     0, e.color,      e.resetColor     },
             { Pid::OFFSET,    0, e.offset,     e.resetOffset    },
             { Pid::AUTOPLACE, 0, e.autoplace,  e.resetAutoplace },
             { Pid::MIN_DISTANCE, 0, e.minDistance, e.resetMinDistance },
             };
-      pList = { { e.title, e.panel } };
+      const std::vector<InspectorPanel> ppList = { { e.title, e.panel } };
+      mapSignals(iiList, ppList);
```

Comment:
1) It is better to make unrelated changes in separate pull requests, or at least in separate commits.
2) This `mapSignals()` call makes each of the related slots (`valueChanged`, `setStyleClicked`) be called at least twice (as each connection gets established that many times) which is not really good. With its current implementation `mapSignals()` should only be called once per `InspectorBase` instance.

Reply:
So are you suggesting the changes to `inspectorElementBase.cpp` is unnecessary?

Reply:
This change actually has unwanted side effects, I don't know whether other similar changes do (although I don't see why they are necessary as well).

Reply:
The changes are necessary to make the inspectors collapsible. I would instead suggest eliminating the `mapSignals()` call in `inspectorBase` to avoid calling it twice.

Reply:
I agree. Will change later.